src/server/cea_608_encoder/cea_608_utility.py

```diff
@@ -56,19 +56,74 @@ def bytes_to_byte_pairs(byte_list: list) -> list:
     return byte_pairs
 
 
-def create_byte_pairs(caption_string: str) -> list:
+def which_char_set(caption_char: str) -> str:
+    """Finds which character set the letter is in
+
+    :param caption_char:
+    :return: the name of the char set the caption_char is in
+    """
+    if caption_char in char_sets['basic_na_set']:
```

Comment:
Since char sets is a dictionary, python allows you to iterate through them like a for-loop.

See the answer here: https://stackoverflow.com/questions/3294889/iterating-over-dictionaries-using-for-loops